camo.py

```diff
@@ -11,20 +11,17 @@ def __init__(self, server, key):
         self.key = key
 
     def image_url(self, url):
-        return self.server + Image(url, self.key).path
-
-    def _rewrite_url(self, url):
         # skip images that have already been proxied and
         # images with data URIs
-        if url.startswith(self.server) or url.startswith("data:image/"):
+        if url.startswith(self.server):
             return url
         else:
-            return self.image_url(url)
+            return self.server + Image(url, self.key).path
 
     def _rewrite_image_urls(self, node):
         for img in node.xpath('.//img'):
             if img.get('src'):
-                img.set('src', self._rewrite_url(img.get('src')))
+                img.set('src', self.image_url(img.get('src')))
         return node
 
     def parse_html(self, string):
```

Comment:
We're no longer using this right? If I'm understanding this correctly, we're now taking advantage of the parsing that happens in bleach, by hooking up a custom filter (i.e. `ImgProxyFilter`).  It's probably fine to leave it here since this is a fork, just wanted to make sure I'm understanding the changes.